File: ex07/ex07_02/ex07_02.c
#include <stdio.h>

int main() {
  FILE *pf;
  char file_name[256] = "../data/ex07_01.txt";
  char line[256];

  pf = fopen(file_name, "r");
  if (pf == NULL) {
    printf("Cannot open file %s\n", file_name);
    return 1;
  }

  while (1) {
    fscanf(pf, "%s", line);
    if (feof(pf)) break;
    printf("%s\n", line);
  }

  fclose(pf);
  return 0;
}

File: ex07/ex07_02/ex07_02.js
import fs from 'fs';
function main() { 
  const filename = "../data/ex07_01.txt";
  fs.readFile(filename, (err, data) => {
    if (err) {
      console.log(`Cannot open file ${filename}`);
      return 1;
    }
    console.log(data.toString());
  });
  return 0;
}
main();